Gemfile.lock

```diff
@@ -719,7 +719,7 @@ DEPENDENCIES
   xmlrpc
 
 RUBY VERSION
-   ruby 2.7.2p137
+   ruby 2.7.5p203
```

Comment:
```suggestion
  ruby 2.7.5p203
```

Reply:
Yeah, bundler... will keep putting it back; so kind of just ignoring it.